pkg/accounting/db.go

```diff
@@ -25,7 +25,15 @@ type Raw struct {
 	CreatedAt       time.Time
 }
 
-//Rollup mirrors dbx.AccountingRollup, allowing us to use that struct without leaking dbx
+// BW mirrors dbx.StoragenodeBandwidthRollup, allowing us to use the struct without leaking dbx
+type BW struct {
+	NodeID        storj.NodeID
```

Comment:
any chance we can call this struct something more descriptive? `StoragenodeBandwidthRollup` or something?